src/compas_rhino/geometry/__init__.py

```diff
@@ -94,5 +96,6 @@
     'RhinoSurface',
     'RhinoVector',
 
-    'RhinoNurbsCurve'
+    'RhinoNurbsCurve',
+    'RhinoNurbsSurface',
```

Comment:
Sorry my first time looking at this part. I'm a bit confused here `RhinoSurface` and `RhinoNurbsSurface` are very similarly named, and they are more or less both wrappers of Rhino geometry instances, however they have quite different kind of APIs, what's the designed context for each of these classes?

Reply:
`RhinoSurface` is a backwards compatibility import from `compas_rhino.conversions` that allows Rhino surface objects and geometries to be converted to compas equivalents.

`RhinoNurbsSurface` is the Rhino plugin implementation of `compas.geometry.NurbsSurface`

Reply:
I see!